algebra/src/curves/bls12_377/mod.rs

```diff
@@ -1,3 +1,15 @@
+//! [Zexe's BLS12-377](https://eprint.iacr.org/2018/962.pdf), the outer curve of edwards_bls12 and
+//! inner curve of the sw6.
+//! Its a BLS curve with embedding degree 12 and base field size is 377 bits.
+//!
+//! Its security against modern version of the number field sieve (STNFS) is estimated as 125 bit
+//! by [HG 2020](https://eprint.iacr.org/2020/351.pdf).
+
+// The HG 2020 result states an unexpected negligible loss in security compared to the recommendations
+// from [Guillevic 2019](https://eprint.iacr.org/2019/1371) which establishes a base field size
+// of 446 Bit to accomodate that security level.
+// I hope that HG will give us some hints on that.
```

Comment:
I would remove the personal flavor in the comment "I hope that".  And I wouldn't use in any case the word "hope" here. We can make it a note for future check or future action.

Reply:
Yes, I agree.